src/AppInstallerCommonCore/pch.h

```diff
@@ -18,6 +18,7 @@
 #include <wincrypt.h>
 #include <deliveryoptimization.h>
 #include <deliveryoptimizationerrors.h>
+#include <amsi.h>

```

Comment:
I'm not sure if we need it in `pch.h` if it is only used in `Archive.cpp`. But I do see the TODO comment from before about moving it 🤔